packages/core/src/components/graphs/bar-stacked.ts

```diff
@@ -203,16 +124,19 @@ export class StackedBar extends Bar {
 	}
 
 	// Un-highlight all elements
-	handleLegendMouseOut = (event: CustomEvent)  => {
+	handleLegendMouseOut = (event: CustomEvent) => {
 		this.parent.selectAll("path.bar")
 			.transition(this.services.transitions.getTransition("legend-mouseout-bar"))
 			.attr("opacity", 1);
 	}
 
 	addEventListeners() {
+		const options = this.model.getOptions();
+		const { groupIdentifier } = options.data;
+
 		const self = this;
 		this.parent.selectAll("path.bar")
-			.on("mouseover", function(datum) {
+			.on("mouseover", function (datum) {
```

Comment:
i noticed that too, i believe there should **not** be one because the linter has it set to never for all function cases 

_tslint.json_
`    "space-before-function-paren": [
      true,
      "never"
    ],`

Reply:
that was recently added to the library, so it could be that I hadn't yet pulled down that `tslint` update before pushing this